test/library/packages/LinearAlgebra/correctness/testMatPowNeg.chpl

```diff
@@ -0,0 +1,15 @@
+use LinearAlgebra;
```

Comment:
A few additional test cases to consider:

- What happens if power is `-1`?
- What happens if matrix is non-invertible?

Reply:
Motivated  by [numpy test suite](https://github.com/numpy/numpy/blob/v1.17.0/numpy/linalg/tests/test_linalg.py#L978-L1060).